core/src/main/scala/kafka/admin/ReassignPartitionsCommand.scala

```diff
@@ -930,6 +935,38 @@ object ReassignPartitionsCommand extends Logging {
     (brokerListToReassign, topicsToReassign)
   }
 
+  /**
+   * The entry point for --alter-throttles. At least one throttle value must be provided.
+   *
+   * @param admin The Admin instance to use
+   * @param interBrokerThrottle The new inter-broker throttle or -1 to leave it unchanged
+   * @param logDirThrottle The new alter-log-dir throttle or -1 to leave it unchanged
+   */
+  def alterThrottles(admin: Admin,
+                     interBrokerThrottle: Long,
+                     logDirThrottle: Long): Unit = {
+    if (interBrokerThrottle < 0 && logDirThrottle < 0) {
+      throw new TerseReassignmentFailureException("No valid throttle values provided to --alter-throttle")
```

Comment:
It would be good to include the flags needed to pass in a throttle in this message